src/drivers/planetscale.ts

```diff
@@ -0,0 +1,80 @@
+import { defineDriver } from "./utils";
+import type { ExecutedQuery } from "@planetscale/database";
+import { connect } from "@planetscale/database";
+import { fetch } from "ofetch";
+
+export interface PlanetscaleDriverOptions {
+  url?: string;
+  table?: string;
+}
+
+interface TableSchema {
+  id: string;
+  value: string;
+  created_at: Date;
+  updated_at: Date;
+}
+
+export default defineDriver((opts: PlanetscaleDriverOptions = {}) => {
+  if (!opts.url)
+    throw new Error("Database URL is required to use the Planetscale driver.");
+
+  opts.table = opts.table || "storage";
+
+  const c = connect({
```

Comment:
Perf: Can we make connection lazy same as other drivers?

Reply:
It is lazy. This only configures the connection.

Reply:
See see checking bundle code. Very confusing naming! What do you think to directly use `new Connection()` here for code clarity that it is not initiating a connection? 

(BTW it _might_ do sync operations in future versions of library or at least try a background connect open init. explicit lazy getter ensures this won't happen)

Reply:
I think using `new Connection()` would be just as likely to perform sync operation in future. We could add a comment if you think it will help 😆

FWIW, I don't think it is likely as the purpose of this library is specifically for use in edge functions: https://github.com/planetscale/database-js.

Reply:
The wrapper is much better IMO to avoid the risks if you don't mind to add. but if prefer to keep like this, also a comment would help so at least anyone reading this driver code as example quickly understands this is not making any connection. `connect()` is basically wrapper for `new Connection`

Reply:
I have added the comment already. Feel free to add the lazy wrapper but I would personally prefer not.